### PR TITLE
improvements and corrections

### PR DESCRIPTION
**Pull request by eltrevii**
i changed some things and corrected other things

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,52 +9,77 @@ A Fabric mod designed to improve server performance at high playercounts.
 **VMP is still in early development and things may break. Please report any issues to our issue tracker.**
 
 ## So what is VMP? 
-Very Many Players, or VMP for short, is a Fabric mod designed to improve general server performance at high playercount 
+Very Many Players (VMP for short) is a Fabric mod designed to improve general server performance at a high player count
```
Comment (ishland) [inline README.md:12]: I personally don't think changing this line is a good idea.

File: README.md [hunk 2]
```diff
@@ -9,52 +9,77 @@ A Fabric mod designed to improve server performance at high playercounts.
...
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+```
+_It can also be read [here](/LICENSE)_
```
Comment (ishland) [inline README.md:85]: Copying license into README seems pretty unnecessary and the original README already have a link to the license.

File: README.md [hunk 3]
```diff
@@ -9,52 +9,77 @@ A Fabric mod designed to improve server performance at high playercounts.
...
-**Server-side game logic performance improvements:**  
+**Server-side game logic performance improvements:**
 - Uses area maps to optimize nearby packet sending and player lookups
 - Uses cache to optimize entity trackers, fluid state lookups, ingredient matching and biome lookup
-- Optimizes natural spawning with caches and other tricks to reduce the amount of work done
+- Optimizes natural spawning with caches and other tricks to reduce the amount of load
 - Optimizes entity tracking with area maps
 - Optimizes entity iteration for collisions
-- Optimizes ticket propagator using MCUtil from the Paper project (patch licensed under MIT)
+- Optimizes ticket propagator using MCUtil from PaperMC (patch licensed under MIT)
```
Comment (ishland) [inline README.md:26]: `the Paper project` is more accurate than `PaperMC` as it also includes Velocity, Waterfall, Starlight and others.

File: README.md [hunk 4]
```diff
@@ -9,52 +9,77 @@ A Fabric mod designed to improve server performance at high playercounts.
...
+## How does VMP achieve performance improvements?
 
 *This list may contain features currently unreleased and only found in development builds*
 
-**Server-side game logic performance improvements:**  
+**Server-side game logic performance improvements:**
 - Uses area maps to optimize nearby packet sending and player lookups
 - Uses cache to optimize entity trackers, fluid state lookups, ingredient matching and biome lookup
-- Optimizes natural spawning with caches and other tricks to reduce the amount of work done
+- Optimizes natural spawning with caches and other tricks to reduce the amount of load
```
Comment (ishland) [inline README.md:23]:
```suggestion
- Optimizes natural spawning with caches and other tricks
```

**Review by ishland — CHANGES_REQUESTED** [review]
Looks good in general but still have some small issues.

**Comment by eltrevii** [comment]
im glad i was useful, i thought i was a piece of 💩

**Pull request closed without merging**